submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,177 @@
+const url = 'https://randomuser.me/api/?results=12';
+
+let aToZLast = document.querySelector('.sortedAtoZLast');
+let zToALast = document.querySelector('.sortedZtoALast');
+let atoZFirst = document.querySelector('.sortedAtoZFirst');
+let ztoAFirst = document.querySelector('.sortedZtoAFirst');
+let content = document.querySelector('.articleBlock');
+let reset = document.querySelector('.reset');
+let male = document.querySelector('#male');
+let woman = document.querySelector('#female');
+let all = document.querySelector('#all');
+let search = document.querySelector('#searchBar');
+let minNumber = document.querySelector('#minNumber');
+let maxNumber = document.querySelector('#maxNumber');
+
+let basicArr1 = [];
+let resultArr = [];
+
+async function getResponse(url) {
+    let response = await fetch(url);
+    let friendsArr = await response.json();
+
+    saveUsers(friendsArr.results, resultArr);
+
+    renderAllItemsToPage(resultArr);
+}
+
+
+function saveUsers(arr, resArr) {
```

Comment:
What does this function do, exactly?
It seems like it just copies the whole `arr` to `resArr` and then does the same for `resultArr` and `basicArr1`. It's not readable and seems like it doesn't make any sense at all. You can just save users right where you get your response from API.